Makefile

```diff
@@ -18,7 +18,7 @@ build-test:
 	docker build -f Dockerfile.test -t shpkpr-test .
 
 # User-defined function to allow easy running of our tests inside Docker
-docker-test = docker run -i -v `pwd`:/src:ro $(1) --rm shpkpr-test $(2)
+docker-test = docker run -i -v `pwd`:/src:ro -v /var/run/docker.sock:/var/run/docker.sock $(1) --rm shpkpr-test $(2)
```

Comment:
What's the reasoning on this change? :+1:

Reply:
`shpkpr run` launches a container in which to run a command. shpkpr's tests run inside a Docker container already, this allows code inside the container to interact with the Docker API thus allowing us to test `shpkpr run` properly by spinning up a new container.

This isn't docker-in-docker, it just allows code inside the container to interact with the host's Docker daemon via HTTP over the socket.